cloud/lib/api-stack.ts

```diff
@@ -104,6 +134,57 @@ export class ApiStack extends Stack {
 		});
 		*/
 
-		fargateService.targetGroup.configureHealthCheck({ path: '/health' });
+		// Create an HTTP APIGateway with a VPCLink integrated with our load balancer
+		const securityGroupName = generateResourceName('vpclink-sg');
+		const vpcLinkSecurityGroup = new SecurityGroup(this, securityGroupName, {
+			vpc,
+			securityGroupName,
+			allowAllOutbound: false,
+		});
+		vpcLinkSecurityGroup.connections.allowFromAnyIpv4(
+			Port.tcp(80),
+			'APIGW to VPCLink'
+		);
+		vpcLinkSecurityGroup.connections.allowTo(
+			fargateService.loadBalancer,
+			Port.tcp(80),
+			'VPCLink to ALB'
+		);
```

Comment:
SecurityGroups are what allow/disallow access to resources inside a VPC (virtual private cloud).

VPCs come with a default security group allowing pretty much everything inbound and outbound, so for obvious reasons those rules are removed (CDK does this for us, which is nice). Annoyingly, the default security group itself cannot be removed, just the rules 🙄 

Here we add a security group to our VPCLink, to allow inbound TCP traffic on port 80 (from API Gateway) and outbound traffic to our Application Load Balancer only.